pkg/kv/kvnemesis/validator_test.go

```diff
@@ -1807,6 +1827,16 @@ func TestValidate(t *testing.T) {
 				rd(k1, k2, t2, s2),
 			),
 		},
+		{
```

Comment:
Is it worth pairing this with at least one or two other interactions with other commands? I've seen the verifier code and the changes are fairly trivial so we probably don't need too much else.

Reply:
Yeah, may as well. Added some shadowing and scans.